crates/engine/src/ext.rs

```diff
@@ -241,32 +241,26 @@ impl Engine {
     }
 
     /// Returns the decoded contract storage at the key if any.
-    pub fn get_storage(&mut self, key: &[u8], output: &mut &mut [u8]) -> Result {
+    pub fn get_storage(&mut self, key: &[u8]) -> core::result::Result<&[u8], Error> {
```

Comment:
Can't you just import the full path?

Reply:
There was a type alias here `type Result = core::result::Result<(), Error>;`. However I've just removed that and replaced its usages with the explicit type.